### PR TITLE
Image or title refer to article

### LINKED ISSUE #1: Feat: Clicking post image or title should ref to it's article.

### PR DESCRIPTION
**Pull request by orestischaral**

### CODE REVIEW

File: components/ArticleList.vue [hunk 1]
```diff
@@ -36,6 +45,7 @@
               </div>
             </div>
           </div>
+          </nuxtlinkv-if="article.img">
```
Comment (MrNaif2018) [inline components/ArticleList.vue]: This was probably a typo, right?

**Review by MrNaif2018 — CHANGES_REQUESTED** [review]
Thank you very much! Works perfectly! Please take a look at review comments.
Also not sure about favicon.ico diff-did it change? Can't get what exactly changed in it.

File: components/ArticleList.vue (hunk 1 above)
Comment (orestischaral) [reply components/ArticleList.vue]: Yes it was!

**Comment by orestischaral** [comment]
Didn't change sth on the favicon. Probably kept from previous branch. It's the same with the white background

**Review by MrNaif2018 — APPROVED** [review]
Thank you very much! Merging!

**Comment by MrNaif2018** [comment]
A tip for the future: you probably have your git email misconfigured, that's why your commits are shown from not your github account, but "anonymous" account. If your email on github and in git will match it will show correctly

**Pull request merged**